src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.State.cs

```diff
@@ -226,17 +223,13 @@ private sealed class FinalState : State
                     HasSuccessfullyLoaded = hasSuccessfullyLoaded;
                     FinalCompilation = finalCompilationSource;
 
-#if DEBUG
```

Comment:
undo?

Reply:
It was messing up the Add Using feature (broke sorting), but I can move it to another PR if you want.

Reply:
Why was this breaking sorting? This feels like we're working around a bug here.

Reply:
➡️ This change has been moved to #45464